### ISSUE TITLE
Add examples of using BIOM executable to the website

### ISSUE THREAD

**Issue by colinbrislawn** [biocore/biom-format#735]
Thanks to #3, there are excellent examples of using the `biom` API. There are also excellent examples in the manpage, but not on the website!

For example, the command `biom normalize-table` is well documented in the manpage, but is [never mentioned on the website](https://www.google.com/search?q=site%3Abiom-format.org+normalize). Because the website is accessible than the manpage for new linux users, I suggest we take the excellent documentation in the manpage, and put it somewhere online.

Where on the website can we feature the examples in the manpage?

**Comment by wasade** [comment]
There's a manpage?

@colinbrislawn I am strongly in favor of expanding the accessible examples and documentation, and would eagerly accept PRs on this

**Comment by colinbrislawn** [comment]
Maybe it's not called a manpage... 

`biom normalize-table --help` provides clear, concise documentation including Usage, Examples, and Option. I really wish this was also online.

The documentation is already really strong. We just need a good place to put it.

**Comment by wasade** [comment]
I think the best long term solution would be to at release, dynamically create a page, per command, based off the docstrings within the commands. Would you be interested in taking point on that?

**Comment by colinbrislawn** [comment]
I agree that reusing the CLI documentation and generating it dynamically would be excellent. While Click has great support for CLI help, I'm not sure how best to pull that into a format which could be rendered on the `gh-pages` static site. This is a little beyond me...
